assets/src/blocks/Media/MediaElementVideo.js

```diff
@@ -0,0 +1,60 @@
+import { useEffect } from '@wordpress/element';
+import { useScript } from '../../components/useScript/useScript';
+import { useStyleSheet } from '../../components/useStyleSheet/useStyleSheet';
+
+export const MediaElementVideo = ({ videoURL, videoPoster }) => {
+
+	const [stylesLoaded, stylesError] = useStyleSheet(
+    `https://cdn.jsdelivr.net/npm/mediaelement@4.2.16/build/mediaelementplayer-legacy.min.css`
+	);
+
+	const [scriptLoaded, scriptError] = useScript(
+		`https://cdn.jsdelivr.net/npm/mediaelement@4.2.16/build/mediaelement-and-player.min.js`
+	);
+
+	const [vimeoRendererLoaded, vimeoRendererError] = useScript(
+		`https://cdn.jsdelivr.net/npm/mediaelement@4.2.16/build/renderers/vimeo.min.js`
```

Comment:
Shouldn't we try to only load this if it's actually a vimeo video?

Reply:
Indeed, but now that you mention it, this script originally added by Wordpress, because Vimeo allowed to link .directly to the .mp4 source of your video, but I'm not sure that's the case anymore, I have some videos there, I'll do a quick check.

Reply:
After double checking with a Vimeo video, I couldn't find any reason why this was needed in the first place. Vimeo links go through the oEmbed part of the code, you can link directly to the MP4 source of the file in Vimeo but the URL is a download link, it does not end with `.mp4` so it won't be used in this caso, and if it were, it's just a regular MP4 file. So I deleted this dependency.